src/pages/low/hardware_topics/electronic-design.md

```diff
@@ -22,3 +23,12 @@ Luos' RS485 official connector is: <a href="https://octopart.com/df11-8dp-2ds%28
 Luos manages any type of microcontrollers, but they need to be added manually to the library. If your microcontroller is not supported yet, please contact us:
  - by mail: <a href="mailto:hello@luos.io">hello@luos.io</a>
  - on <a href="https://github.com/Luos-io/Luos/issues/new?assignees=nicolas-rabault&labels=porting&template=porting-request.md&title=%5BMCU+PORTING%5D+" target="_blank">GitHub</a>
+
+## PTP
```

Comment:
A short sentence of introduction of what is PTP and what it does is missing

Reply:
done